code/index.html

```diff
@@ -3,28 +3,97 @@
+        <h3>Request your virtual assistance</h3>
+        <label>
+          Name:<input name="form-name" type="text" required></input>
+        </label>
+        <p>What kind of virtual assistant service are you interested in?</p>
+        <label>
+          Content writing <input type="checkbox" value="content" name="servicechoice"></label></input>
+        </label>
+        <label>
+          Social Media <input type="checkbox" value="some" name="servicechoice"></label></input>
+        </label>
+        <label>
+          Customer service <input type="checkbox" value="customerservice" name="servicechoice"></label></input>
+        </label>
+        <label>
+          Other <input type="checkbox" value="otherservices" name="servicechoice"></label></input>
+        </label>
+        <p>Would you like help on a monthly basis or one time help?</p>
+        <label>
+          Monthly basis <input type="radio" value="monthly" name="packagetype" required></label></input>
+        </label>
+        <label>
+          One time <input type="radio" value="onetime" name="packagetype" required></label></input>
+        </label>
+        <label>
+          E-mail: <input type="email" name="form-email" required></input>
+        </label>
+        <button type="submit" value="Submit" class="form-button">Submit</button>
+      </form>
+    </div>
 
     <section class="accordion">
-      <!-- Accordion -->
       <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="accordion-container">
+        <div>
+          <h3 id="accordion-header-1" class="accordion-header">Can you do translations?</h3>
+          <p id="accordion-content-1" class="accordion-content">
+            Yes, I can translate from English, Norwegian and Danish to Swedish.
+          </p>
+        </div>
+        <div>
+          <h3 id="accordion-header-2" class="accordion-header">Can you handle remote meetings?</h3>
+          <p id="accordion-content-2" class="accordion-content">
+            Yes, I have experience from using Teamviewer, Teams, Zoom and Scopia and can gladly give other tools a try too.
+          </p>
+        </div>
+        <div>
+          <h3 id="accordion-header-3" class="accordion-header">
+            Can you help with project management in an agile team?
+          </h3>
+          <p id="accordion-content-3" class="accordion-content">
+            Yes, I have experience from agile methods and have a certificate in Prince2 Agile Practitioner.
+          </p>
+        </div>
       </div>
+    </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+    <footer class="footer">
+      <div class="footer-text">
+        <p>© Therese Hagelin
+          <br>
+          <br>
```

Comment:
We can solve this with CSS instead! Divide your text into three `p` and give them margin :)

Reply:
Lovely, fixed now. Thanks a lot for a great advice.